Makefile

```diff
@@ -102,6 +105,13 @@ gen-openapi:
 	$(PROTOC) --swagger_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/collector/metrics/v1/metrics_service_http.yaml:$(PROTO_GEN_OPENAPI_DIR) opentelemetry/proto/collector/metrics/v1/metrics_service.proto
 	$(PROTOC) --swagger_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/collector/logs/v1/logs_service_http.yaml:$(PROTO_GEN_OPENAPI_DIR) opentelemetry/proto/collector/logs/v1/logs_service.proto
 
+# Generate gRPC/Protobuf implementation for Parquet.
+.PHONY: gen-parquet
+gen-parquet:
+	rm -rf ./$(PROTO_GEN_PARQUET_DIR)
+	mkdir -p ./$(PROTO_GEN_PARQUET_DIR)
+	$(foreach file,$(PROTO_FILES),$(call exec-command, $(PROTOC) --parquet_out=go_file=true$(comma)output_messages=TracesData$(semicolon)LogsData$(semicolon)MetricsData$(comma)max_recursion=3:./$(PROTO_GEN_PARQUET_DIR) $(file)))
```

Comment:
This "comma" and "semicolon" trick can be done consistent across all gen and in a separate PR.

Reply:
That's just needed as a pretty bad crutch for this foreach instruction. There might be a better way around.

Reply:
this crutch is needed right now or the generation fails, so leaving this in for now.